processor/workers.go

```diff
@@ -162,24 +173,49 @@ func codeState(
 
 			switch tokenType, offsetJump, endString := langFeatures.Tokens.Match(fileJob.Content[i:]); tokenType {
 			case TString:
+				// TODO if we are in string state then check what sort of string so we know if docstring OR ignoreescape string
+				// TODO need to jump over the end of the match as @" will enter, then exit straight away
+				ignoreEscape := false
+
+				// loop over the string states and if we have the special flag match, and if so we need to ensure we can handle them
+				for k := 0; k < len(langFeatures.Quotes); k++ {
+					if langFeatures.Quotes[k].DocString || langFeatures.Quotes[k].IgnoreEscape {
+						// If so we need to check if where we are falls into these conditions
+						isMatch := true
+						for j := 0; j < len(langFeatures.Quotes[k].Start); j++ {
+							if fileJob.Content[index+j] != langFeatures.Quotes[k].Start[j] {
+								isMatch = false
```

Comment:
We could `break` out of the loop here. Once we've set `isMatch = false` there's no purpose in continuing the loop.

Reply:
Done.